editor/groups_editor.cpp

```diff
@@ -196,22 +208,20 @@ void GroupDialog::_add_group_pressed(const String &p_name) {
 	add_group_text->clear();
 }
 
-void GroupDialog::_add_group(String p_name) {
+void GroupDialog::_add_group(String p_name, bool can_be_edited) {
```

Comment:
Parameters which are only used, but not written to, should start with `p_`.
Parameters which are written to should start with `r_` (relevant below).